Makefile

```diff
@@ -764,3 +764,32 @@ include tests/integration/tests.mk
 .PHONY: benchcheck
 benchcheck:
 	bin/perfcheck.sh
+
+#-----------------------------------------------------------------------------
+# Target: istio-ecosystem installer
+#-----------------------------------------------------------------------------
+
+export ISTIO_INSTALLER?=${GOPATH}/src/github.com/istio/installer
+INSTALLER_OPTS="-f install/kubernetes/helm/istio/test-values/values-installer-e2e.yaml --set global.istioNamespace=istio-system --set global.hub=${HUB} --set global.tag=${TAG}"
+INSTALLER_REPOSRC=git://github.com/istio/installer.git
+
+force: ;
+
+${GOPATH}/src/github.com/istio/installer: force
+	mkdir -p ${GOPATH}/src/istio-ecosystem
+	git clone -b master "${INSTALLER_REPOSRC}" "${ISTIO_INSTALLER}" 2> /dev/null || (cd "${ISTIO_INSTALLER}" ; git pull)
+
+generate_installer_e2e_yaml: ${GOPATH}/src/github.com/istio/installer istio-init.yaml
+	rm -f install/kubernetes/istio-installer.yaml
+	for f in ${ISTIO_INSTALLER}/crds/files/*.yaml; do (cat $$f; echo '---') >> install/kubernetes/istio-installer.yaml; done
```

Comment:
is this `$$f` right?  won't `$$` give the pid anf then the literal `f`?

Reply:
this works in all of the environments that I've tested it in...

Reply:
Oh I guess maybe in the makefile `$$` escapes? Since directly in bash gives:
```
$ for f in crds/files/*.yaml; do (cat $$f; echo '---'); done
cat: 114134f: No such file or directory
---
cat: 114134f: No such file or directory
---
cat: 114134f: No such file or directory
---
cat: 114134f: No such file or directory
---
cat: 114134f: No such file or directory
---
```

If it works that is probably what is happening